usage-insights-launch.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Introducing Usage Insights
```

Comment:
```suggestion
title: Introducing usage monitoring
```

Since we don't provide any insights, I've been referring to it as usage monitoring. Only because I'd like to eventually release insights on our usage haha. Let me know what you think about the swap